docs/extensions/tutorials/_posts/1970-01-05-Installing3rdPartyPackages.md

```diff
@@ -0,0 +1,323 @@
+```
+
+Opening the SwiperApp in the Builder will show us an app with no Screens, but since we just installed our Swiper Extension onto the app, we can just add the Swiper Screen to it by clicking on the + button next to Screens, going to the Custom category and selecting the Swiper Extension. As soon as it's loaded into the Main Navigation we can preview the app:
+
+<p class="image">
+<img src='{{ site.url }}/img/tutorials/installing-3rd-party-and-native-packages/swiper-in-preview.png'/>
+</p>
+
+## 2) Installing a Native Package
+
+When installing a 3rd party package that uses native functionalities we have to make sure we link the native dependencies it has. This is done using postlink scripts. As an example, we'll be making a QR Code reader that's going to display what the scanned QR code says. To scan a QR code we'll need to use the devices camera, which we'll get access to using react-native-camera, a 3rd party package for utilizing device cameras.
+
+### Making an Extension
+
+To begin, we'll need an extension, so let's just make one. If this is something you don't know how to do yet, you should really go through [My First Extension]({{ site.url }}/docs/extensions/my-first-extension/introduction) to make sure you understand the fundamentals.
+
+We begin by initializing an extension and writing in the basic information about it.
+
+```ShellSession
+$ shoutem init qr-reader-extension
+Enter information about your extension. Press `return` to accept (default) values.
+Title: QR Reader
+Version: 0.0.1
+Description: An extension that scans QR codes and displays the encoded information to the user.
+Initializing extension:
+Installing packages for server...
+...
+
+Extension initialized.
+```
+
+Let's switch over to the extension directory and add a screen with a shortcut that will be the user interface for our QR Reader.
+
+```ShellSession
+$ cd qr-reader-extension
+$ shoutem screen add QRReaderScreen --shortcut QRReader
+Enter shortcut information:
+Title: QR Reader
+
+Screen `QRReaderScreen` created in file `app/screens/QRReaderScreen.js`!
+Shortcut `QRReader` created.
+Shortcut `QRReader` opens `QRReaderScreen` screen.
+File `app/extension.js` was modified.
+File `extension.json` was modified.
+```
+
+### Installing the Package
+
+To make sure the native dependencies are linked, we'll have to make sure our postlink script is run by putting it in our `app/package.json` file.
+
+```json
+#file: app/package.json
+{
+  "name": "{{ site.example.devName }}.qr-reader-extension",
+  "version": "0.0.1",
+  "description": "An extension that scans QR codes and displays the encoded information to the user.",
+  "dependencies": {
+    "react-native-camera": "^0.4.1"
+  },
+  "rnpm": {
+    "commands": {
+      "postlink": "node node_modules/{{ site.example.devName }}.qr-reader-extension/scripts/run.js"
+    }
+  }
+}
+```
+
+We have to make a `scripts` directory and make the postlink script in it: `app/scripts/run.js`
```

Comment:
Use "Create `scripts` directory" instead of "we have to make", because it seems like you need to name it exactly `scripts` directory.